apps/desktop/src/shared/context/contributors/githubIssue.ts

```
@@ -0,0 +1,40 @@
+}
+
+export const githubIssueContributor: ContextContributor<{
+	kind: "github-issue";
+	url: string;
+}> = {
+	kind: "github-issue",
+	displayName: "GitHub Issue",
+	description: "Full issue body fetched and inlined as context.",
+	requiresQuery: true,
+	async resolve(source, ctx) {
+		let issue;
```

Biome (2.4.11)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)